src/Inventory/InventoryList.js

```diff
@@ -127,35 +105,71 @@ class InventoryList extends Component {
 		this.setState({ newItemInputVisible: false, name: '' });
 	}
 
+	renderItem(inventories) {
+		const listId = this.props.navigation.state.params.id;
+		const rightButtons = [
+			{
+				text: (<Icon
+					name='edit'
+					color={colors.tint}
+					size={24}
+					underlayColor='transparent'
+				/>),
+				backgroundColor: 'orange',
```

Comment:
Should we put some of these styles in `styles.js` so we have consistent coloring for the swipe stuff?